### DISCUSSION TITLE
How Do I Import an Observation List

### DISCUSSION THREAD

**Discussion by erikwilliamsa** [Stellarium/stellarium#3364] [Q&A] [answered]
I can't seem to import observation lists and I'm not sure if I'm doing something wrong.

When I go to import the list, the first time I get no message and nothing is added.    When I retry the same `json` file I get the following message:

![image](https://github.com/Stellarium/stellarium/assets/7417740/95910a8e-62b5-412d-93b1-faa5c220bbe8)

When I click accept nothing happens.

However, if I close and reopen the application, I can select a list with no name/contents:

![image](https://github.com/Stellarium/stellarium/assets/7417740/86a87964-7ab6-4076-99fe-db6f44a6f5c0)

Stellarium 23.2
Version 23.2.0
Based on Qt 6.4.3

Any help would be appreciated!

**Comment by gzotti** [comment, marked as answer]
#3207
